Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Copyright 2022 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Build the binary
+FROM golang:1.18 AS builder
+
+WORKDIR /build-dir
+# Copy the go source
+COPY . .
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+USER 0
+RUN go mod download
+RUN make build
+
+FROM quay.io/centos/centos:stream8
+LABEL maintainer="KCP QE Team"
+USER root
+WORKDIR /
+
+RUN dnf install -y curl unzip jq && \
```

Comment:
Any special reason to involve curl? quay.io/centos/centos:stream8 includes it, dnf install -y curl will update it but any special reason to update it?

Reply:
Good point. Previously I used apline it doesn't have curl, but it seems miss other libs the kcp plugins not worked,  I think update the curl is no harm in centos:stream8. Anyway we could optimize it without extra useless steps.

Reply:
OK you can keep it. Just curious : )